src/main/java/fr/openent/lystore/service/impl/DefaultInstructionService.java

```diff
@@ -34,6 +38,8 @@ public class DefaultInstructionService  extends SqlCrudService implements Instru
 
     private static final Logger log = LoggerFactory.getLogger (DefaultOrderService.class);
     private DefaultOperationService operationService = new DefaultOperationService(Lystore.lystoreSchema, "operation");
+    private OrderService orderService = new DefaultOrderService(Lystore.lystoreSchema, LystoreBDD.ORDER_CLIENT_EQUIPMENT , null);
+    private OrderRegionService orderRegionService = new DefaultOrderRegionService(LystoreBDD.ORDER_REGION_EQUIPMENT);
```

Comment:
Ils peuvent être final 

```suggestion
    private final OrderService orderService = new DefaultOrderService(Lystore.lystoreSchema, LystoreBDD.ORDER_CLIENT_EQUIPMENT , null);
    private final OrderRegionService orderRegionService = new DefaultOrderRegionService(LystoreBDD.ORDER_REGION_EQUIPMENT);
```